tests/conftest.py

```diff
@@ -1,6 +1,16 @@
 import pytest
 import csv
 import ipaddr as ipaddress
+import logging
+
+from ansible_host import ansible_host
+from loganalyzer.loganalyzer import LogAnalyzer
+pytest.disable_loganalyzer = False # Log analyzer will be turned on for all test cases
+
+pytest_plugins = ('ansible_fixtures')
+# Disable debug messages from pytest-ansible lib
+logging.getLogger("pytest_ansible").setLevel(logging.WARNING)
```

Comment:
Don't disable. DEBUG from pytest-ansible is helpful. User can control logging from CLI as pytest allows